avm/res/event-grid/topic/event-subscription/main.bicep

```
@@ -54,7 +54,7 @@ resource eventSubscription 'Microsoft.EventGrid/topics/eventSubscriptions@2022-0
     deadLetterDestination: !empty(deadLetterDestination) ? deadLetterDestination : null
     deadLetterWithResourceIdentity: !empty(deadLetterWithResourceIdentity) ? deadLetterWithResourceIdentity : null
     deliveryWithResourceIdentity: !empty(deliveryWithResourceIdentity) ? deliveryWithResourceIdentity : null
-    destination: destination
+    destination: !empty(deliveryWithResourceIdentity) ? null: destination
```

```suggestion
    destination: empty(deliveryWithResourceIdentity) ? destination : null
```
Same as the other comment :)

---

OK !